lib/relations/Relation.js

```diff
@@ -347,24 +355,12 @@ function createRelationProperty(ctx, refString, propName) {
 function parseModify(ctx) {
   const mapping = ctx.mapping;
   const modifier = mapping.modify || mapping.filter;
-  let modify;
-
-  if (modifier) {
-    try {
-      modify = createModifier({
-        modifier,
-        modelClass: ctx.relatedModelClass
-      });
-    } catch (err) {
-      if (err instanceof ModifierNotFoundError) {
-        throw ctx.createError(err.message);
-      } else {
-        throw err;
-      }
-    }
-  } else {
-    modify = () => {};
-  }
+  const modify =
+    modifier &&
+    createModifier({
+      modifier,
+      modelClass: ctx.relatedModelClass
+    });
```

Comment:
@koskimas not sure you like writing it like this?

Reply:
That's fine.